Description: Don't use nautilus-classic since it forces desktop icons
Author: Jeremy Bicha <jbicha@ubuntu.com>
Bug: https://bugzilla.gnome.org/695088

Index: gnome-shell-extensions-3.15.4/data/gnome-classic.session.desktop.in.in
===================================================================
--- gnome-shell-extensions-3.15.4.orig/data/gnome-classic.session.desktop.in.in
+++ gnome-shell-extensions-3.15.4/data/gnome-classic.session.desktop.in.in
@@ -1,3 +1,3 @@
 [GNOME Session]
 _Name=GNOME Classic
-RequiredComponents=gnome-shell;gnome-settings-daemon;nautilus-classic;
+RequiredComponents=gnome-shell;gnome-settings-daemon;
